game/scripts/npc/abilities/broodmother_spin_web.txt

```diff
@@ -38,7 +38,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heath_regen"                                     "3 6 9 12 22 35" // OAA
+        "heath_regen"                                     "6 12 18 24 36 72" // OAA
```

Comment:
> Reduce this to 5/10/15/20/35/50.

I find these changes good but I will change everything if that is ok. I need to have a really good reason to nerf the hp regen buff because hp regen (especially later on is kinda useless)

Reply:
Ok, leave it like this then.